dockerclient.go

```diff
@@ -97,6 +97,41 @@ func (client *DockerClient) doRequest(method string, path string, body []byte, h
 	return data, nil
 }
 
+func (client *DockerClient) doStreamRequest(method string, path string, in io.Reader, headers map[string]string) (io.ReadCloser, error) {
```

Comment:
doStreamRequest: the input and output is using io.Reader and io.Writer. it can work streamly.
doRequest: the input and output is byte[]. if we send a request, we have to wait all the response. it can not work streamly.